.github/workflows/create-draft-release.yml

```diff
@@ -1,6 +1,10 @@
 name: Create Draft Release
 on:
   workflow_dispatch:
+    inputs:
+      branch_name:
```

Comment:
Can we refactor the two inputs to be release number instead of branch name? Just to make it consistent with the other action? 

version_number:
        description: 'The number for this release in the format: 0.0.0'
        required: true

Reply:
The important thing is that it knows which branch to check out.  Having the user specify the branch name exactly is the simplest, but I bet it's not too hard to have it `grep` the list of tags to find a match for e.g. "1.3.0".  I'll take a look.

Reply:
Actually it looks like if we can rely on the selector menu of the "Use workflow from" section of the action to release the latest commit on the selected branch.  Does that sound okay?

Reply:
Yes a drop-down for the branches sounds good